crates/forge/src/cmd/eip712.rs

```diff
@@ -97,7 +83,13 @@ impl Eip712Args {
             Ok(())
         })?;
 
-        eyre::ensure!(compiler.sess().dcx.has_errors().is_ok(), "errors occurred");
+        // `compiler.sess()` inside of `ProjectCompileOutput` is built with `with_buffer_emitter`.
```

Comment:
```suggestion
        // SAFETY: `compiler.sess()` inside of `ProjectCompileOutput` is built with `with_buffer_emitter`.
```

i find the preamble useful for context

Reply:
SAFETY should only be used for `unsafe`